docs/DEBUGGING_IOS.md

```diff
@@ -0,0 +1,47 @@
+## iOS
+
+You gonna have to build a `main.jsbundle` file to add into Xcode and provide to the app, use the following command:
```

Comment:
Are we sure this is absolutely correct? I am able to just change to "Release" scheme and it works fine.

Is this maybe an XCode 9 only issue?

Reply:
Alright, I run it and it works. But I updated to Xcode 10, so I can't tell if I did something wrong or not before. I'll change it with the "release" switch behavior, and keep the rest if people are still having troubles